force-app/main/default/dashboards/ConstructionDBFolder1.dashboardFolder-meta.xml

```diff
@@ -2,7 +2,7 @@
 <DashboardFolder xmlns="http://soap.sforce.com/2006/04/metadata">
     <folderShares>
         <accessLevel>Manage</accessLevel>
-        <sharedTo>test-mzyso6wots4v@example.com</sharedTo><!--Change to the admin username in the org-->
```

Comment:
run the git update-index --assume-unchanged command for this file so we don't have to have this in our commits anymore